community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java

```diff
@@ -97,11 +97,6 @@
 
     /**
      * Instantiate a graph database given configuration and dependencies.
-     *
```

Comment:
Those javadocs without `@param`s and `@return` are invalid and java 8 doclint will complain. Ideally they should be fixed  and not removed.